### ISSUE TITLE
Unsafe build flags in 8.14.0 - cannot build

### ISSUE THREAD

**Issue by nemecek-filip** [getsentry/sentry-cocoa#3347]
### Platform

iOS

### Installed

Swift Package Manager

### Version

8.14.0

### Steps to Reproduce

Install 8.14.0 with SPM
Start build in Xcode 15

### Expected Result

Builds

### Actual Result

Build error:

```
The package product 'Sentry' cannot be used as a dependency of this target because it uses unsafe build flags.
```

<img width="421" alt="image" src="https://github.com/getsentry/sentry-cocoa/assets/49204469/788bae0a-1582-4c17-8bf4-e46bd44d3caa">

**Comment by nemecek-filip** [comment]
I am sorry for the minimal amount of detail, but this is first time I am seeing build error like this and there isn't anything more specific in Xcode

**Comment by gabors** [comment]
@nemecek-filip I just ran into the very same issue. Also scratching my head :)

**Comment by gabors** [comment]
https://forums.swift.org/t/override-for-unsafeflags-in-swift-package-manager/45273

**Comment by kahest** [comment]
Thanks for reporting, we'll investigate!

**Comment by twprzybysz** [comment]
+1

**Comment by mmaetzler** [comment]
+1, but on version 8.2.0 that still worked to build a few days ago.

**Comment by armcknight** [comment]
I just encountered this myself on an unrelated demo app. A fix is in the works now!

**Comment by armcknight** [comment]
@mmaetzler @nemecek-filip @gabors We just released the hotfix in 8.14.1, please let us know if that gets you back to normal again!

**Comment by mmaetzler** [comment]
I can confirm that 8.14.1 works. Thanks for the quick fix!
